hbase-server/src/test/java/org/apache/hadoop/hbase/master/balancer/TestStochasticLoadBalancer.java

```diff
@@ -284,6 +284,25 @@ public void testLocalityCost() throws Exception {
     }
   }
 
+  @Test
+  public void testMoveCostMultiplier() throws Exception {
+    Configuration conf = HBaseConfiguration.create();
+    StochasticLoadBalancer.CostFunction
+        costFunction = new StochasticLoadBalancer.MoveCostFunction(conf);
+    BaseLoadBalancer.Cluster cluster = mockCluster(clusterStateMocks[0]);
+    costFunction.init(cluster);
+    costFunction.cost();
+    assertEquals(7, costFunction.getMultiplier(), 0.01);
```

Comment:
Not sure if we want to replace 7 with `DEFAULT_MOVE_COST`, which currently is private...same to `DEFAULT_MOVE_COST_OFFPEAK`

Reply:
> Can we please avoid @VisibleForTesting and make them private? On the other hand, use default value directly in test cases. If default values change here and test is not updated, it is anyways going to fail.

Discussed about it above with viraj.

Reply:
@liuml07  Yeah, you are right. Thanks for the comment.

Reply:
@liuml07 @bsglz I think it's fine to use constant directly. No worries, keep them public and use it here.

Reply:
Ok, will fix later. Thanks.